CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 # Changelog for AMD Debugger API (ROCdbgapi)
 
-Full documentatino for AMD Debugger API is available at
-[rocm.docs.amd.com](https://rocm.docs.amd.com/projects/ROCdbgapi/en/latest/index.html).
+Full documentation for AMD Debugger API is available at
+[rocm.docs.amd.com/rocdbgapi](https://rocm.docs.amd.com/projects/ROCdbgapi/en/latest/index.html).
 
 ## rocm-dbgapi-0.77.0
```

Comment:
@lancesix This needs to be replaced with "rocdbgapi version number for ROCm 6.3". Please let me know which headings are applicable to be listed under 6.3 along with the rocdbgapi version number.